package.json

```diff
@@ -34,6 +34,13 @@
 	},
 	"homepage": "https://github.com/balena-io/leviathan-worker#readme",
 	"dependencies": {
+		"@types/bluebird": "^3.5.25",
+		"@types/bluebird-retry": "^0.11.2",
+		"@types/express": "^4.16.1",
+		"@types/firmata": "^0.19.0",
+		"@types/lodash": "^4.14.120",
+		"@types/mz": "0.0.32",
```

Comment:
Are you sure this should be here? It's only necessary when you export the type directly.

Reply:
These should all be devDependencies, except @types/bluebird maybe.